### PR TITLE
Lms 103

### PR DESCRIPTION
**Pull request by MikolaLvivUA**

### CODE REVIEW

File: src/routes/course/course.router.ts [hunk 1]
```diff
@@ -1,16 +1,17 @@
 import { Router } from 'express';
 
 import { courseController } from '../../controllers';
+import { isCoursePresent } from '../../middleware/course';
 
 const router = Router();
 
-router.get('/', courseController.getAllCourses);
+router.get('/', courseController.getCourses);
 router.get('/:course_id', courseController.getCourseById);
```
Comment (VictorKmin) [inline src/routes/course/course.router.ts]: isCoursePresent ?

File: src/routes/course/course.router.ts [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
 
 router.post('/', courseController.createCourse);
 
-router.patch('/:course_id', courseController.editCourse);
+router.patch('/:course_id', isCoursePresent, courseController.editCourse);
```
Comment (VictorKmin) [inline src/routes/course/course.router.ts]: router.use('/:course_id', isCoursePresent) and then this methods w/o middleware

File: src/validators/courseFilterParametresValidator.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import * as Joi from 'joi';
+
+export const filterParametresValidator = Joi.object().keys({
+  label: Joi.string().max(255).trim(),
+  level: Joi.alternatives().try([Joi.string().max(255).trim(), Joi.number()]),
```
Comment (VictorKmin) [inline src/validators/courseFilterParametresValidator.ts]: Check if Interface has 2 types. We have to unification this

Comment (MikolaLvivUA) [reply src/validators/courseFilterParametresValidator.ts]: fixed

File: src/routes/course/course.router.ts (hunk 2 above)
Comment (MikolaLvivUA) [reply src/routes/course/course.router.ts]: fixed

File: src/routes/course/course.router.ts (hunk 1 above)
Comment (MikolaLvivUA) [reply src/routes/course/course.router.ts]: fixed

**Pull request merged**